<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiskImageWidget</class>
 <widget class="QWidget" name="DiskImageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modify Guest Image</string>
  </property>
  <widget class="QPushButton" name="toHostPushButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>330</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="toGuestPushButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>280</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>521</width>
     <height>591</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTreeView" name="hostTreeView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="hostComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>500</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Host</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>10</y>
     <width>521</width>
     <height>591</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTreeView" name="guestTreeView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="guestComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>500</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Guest</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="detailsPushButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>400</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Details</string>
   </property>
  </widget>
  <zorder>frame_2</zorder>
  <zorder>frame</zorder>
  <zorder>toHostPushButton</zorder>
  <zorder>toGuestPushButton</zorder>
  <zorder>detailsPushButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
